StellarApp/Controllers/NewsView.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  NewsView.swift
+//  StellarApp
+//
+//  Created by Ramu on 4/11/19.
+//  Copyright © 2019 TingxinLi. All rights reserved.
+//
+
+import UIKit
+
+class NewsView: UIView {
+
+    lazy var NewsCollectionView: UICollectionView = {
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .vertical
+        let ncv = UICollectionView.init(frame: self.bounds, collectionViewLayout: layout)
+        ncv.register(NewsCollectionViewCell.self, forCellWithReuseIdentifier: "NewsCell")
+        return ncv
+        
+    }()
+    
+    
+    override init(frame: CGRect) {
+        super.init(frame: UIScreen.main.bounds)
+        commonInit()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        commonInit()
+    }
+    
+    private func commonInit() {
+    
+        setConstraints()
+    }
+    
+    func setConstraints() {
+        self.addSubview(NewsCollectionView)
+        NewsCollectionView.backgroundColor = .clear
+        NewsCollectionView.translatesAutoresizingMaskIntoConstraints = false
+        NewsCollectionView.topAnchor.constraint(equalTo: safeAreaLayoutGuide.topAnchor, constant: 11).isActive = true
```

Comment:
It's probably more efficient to just use 
```swift
NSLayoutConstraints.activate([ 
NewsCollectionView.topAnchor.constraint(equalTo: safeAreaLayoutGuide.topAnchor, constant: 11),
         NewsCollectionView.leadingAnchor.constraint(equalTo: safeAreaLayoutGuide.leadingAnchor, constant: 8),
         NewsCollectionView.trailingAnchor.constraint(equalTo: safeAreaLayoutGuide.trailingAnchor, constant: -8),
         NewsCollectionView.bottomAnchor.constraint(equalTo: safeAreaLayoutGuide.bottomAnchor, constant: -11),
])
```

Not a big deal, just seems repetitive to set `isActive` = true for each if there's a method built for that already.